lib/coap/phoenix/conn.ex

```diff
@@ -12,7 +12,8 @@ defmodule CoAP.Phoenix.Conn do
       headers: headers,
       qs: qs,
       peer: {remote_ip, _},
-      owner: owner
+      owner: owner,
+      message: message
```

Comment:
Should we store just the `message.payload` as body.

Reply:
Sure, I started out that way but then saw message was already available.  Happy to change it.